source/compatibility.tex

```diff
@@ -71,6 +71,33 @@
 Valid \CppXX{} code that \tcode{\#include}{s} headers with these names may be
 invalid in this revision of \Cpp{}.
 
+\rSec2[diff.cpp20.concepts]{\ref{concepts}: concepts library}
+
+\diffref{cmp.concept,concept.equalitycomparable,concept.totallyordered}
+\change
+Replace \tcode{common_reference_with} in \tcode{three_way_comparable_with},
+\tcode{equality_comparable_with}, and \tcode{totally_ordered_with}
+with an exposition-only concept.
+\rationale
+Allow uncopyable, but movable, types to model these concepts.
+\effect
+Valid \CppXX{} code relying on subsumption
+with \tcode{common_reference_with}
```

Comment:
Replace these `\tcode` with `\libconcept`, or remove the `\libconcept` in the examples. Are there index entries into [diff]?

Reply:
I'm not sure we have a policy on that. [diff] certainly seems to have some index entries, but I'm not sure if we want to index everything therein. Similarly, sometimes an example is indexed, I believe that has been the historic avenue for "creative" index entries. But we might also want to clean that up a bit. Let's look at this later.